src/test/java/fi/aalto/cs/apluscourses/presentation/MainViewModelUpdaterTest.java

```diff
@@ -102,7 +102,7 @@ public void testGetUpdatableModulesWithNullCourseReturnEmpty() {
   }
 
   @Test
-  public void testGetUpdatableModulesWithGetCourseFileModuleIdsThrowingReturnsEmpty()
+  public void disabledTestGetUpdatableModulesWithGetCourseFileModuleIdsThrowingReturnsEmpty()
```

Comment:
did you mean [`@Ignore`](https://github.com/junit-team/junit4/wiki/Ignoring-tests) or ...?

Reply:
That's convenient, thanks :smile: The course file metadata method doesn't throw anymore so I "disabled" this test